docs/esp32/quickref.rst

```diff
@@ -171,6 +171,30 @@ Notes:
 * The pull value of some pins can be set to ``Pin.PULL_HOLD`` to reduce power
   consumption during deepsleep.
 
+UART (serial bus)
+-----------------
+
+See :ref:`machine.UART <machine.UART>`. ::
+
+    from machine import UART
+
+    uart1 = UART(1, baudrate=9600, tx=33, rx=32)
+    uart1.write('hello')  # write 5 bytes
+    uart1.read(5)         # read up to 5 bytes
+
+The ESP32 has three hardware UARTs. UART0, UART1 and UART2.
+All of the UARTs have default GPIO assigned to them, however depending on your
+ESP32 variant and board, these pins may conflict with onboard peripherals.
+
+Any GPIO can be used for hardware UARTs using the GPIO matrix, so to avoid
+conflicts simply provide ``tx`` and ``rx`` pins when constructing.
+
+UART0 (GPIO1=TX, GPIO3=RX) is dedicated to the REPL.
+
+UART1 (GPIO10=TX, GPIO9=RX) default pins may conflict if your board uses PSRAM.
+
+UART2 (GPIO17=TX, GPIO16=RX) default pins may conflict with embedded flash.
```

Comment:
might be better to make a table with these defaults, to match the SPI and I2C sections

Reply:
Implemented.